hw/ip/otbn/util/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+  echo "Basic script to build a single .S asm file for OTBN. Input file is first"
+  echo "argument, prefix for output files is second argument. Only use .data"
+  echo "and .text sections in input asm"
+  echo ""
+  echo "$0 input.S out_dir/prog"
+  echo ""
+  echo "Will produce:"
+  echo ""
+  echo "* out_dir/prog.o - input.S assembled object file"
+  echo "* out_dir/prog.elf - Linked binary"
+  echo "* out_dir/prog.dis - Disassembled binary"
+  echo "* out_dir/prog_imem.elf - .text section alone"
+  echo "* out_dir/prog_dmem.elf - .data section alone"
+  exit 1
+fi
+
+if [ -z "$OTBN_UTIL_DIR" ]; then
+  SCRIPT_FILE=$(readlink -f "$0")
+  OTBN_UTIL_DIR=$(dirname "$SCRIPT_FILE")
+fi
```

Comment:
I'm not a fan of the check here. Do we have a use case for running with a different set of tools? If not, let's base the path on the script unconditionally.

Also, since this is a bash script, it might be better to use `$BASH_SOURCE` than `$0` (works better with symlinks, IIRC).